zio-http/src/main/scala/zio/http/model/headers/values/Accept.scala

```diff
@@ -0,0 +1,57 @@
+package zio.http.model.headers.values
+
+import zio.http.model.MediaType
+
+/** Accept header value. */
+sealed trait Accept
+
+object Accept {
```

Comment:
Shouldn't we also propagate that to `HeaderInputs` ?

Reply:
Right 🤦‍♂️